coalib/parsing/FilterHelper.py

```diff
@@ -15,7 +15,20 @@ def apply_filter(filter_name, filter_args, all_bears=None):
         raise InvalidFilterException(filter_name)
     if not filter_args or len(filter_args) == 0:
         return all_bears
-    return available_filters[filter_name](all_bears, filter_args)
+
+    filter_function = available_filters[filter_name]
+
+    def filter_section_bears(bears, args):
```

Comment:
please move it to module scope and precede it with an underscore: `def _filter_section_bears`
Not necessary for the function to be inside `apply_filter` scope

Reply:
okay 👍